src/App.tsx

```diff
@@ -34,8 +33,7 @@ class App extends React.Component<any> {
                 <Router>
                     <div>
                     <Route render={() => <Redirect push to={this.state.redirectPath}/>}></Route>
-                    <Route path='/one' component={OneComponent}></Route>
-                    <Route path='/two' component={TwoComponent}></Route>
+                    <Route path='/one' render={() => <Page activeSmartContract="penguinContract@0.0.1"/>}></Route>
```

Comment:
Probably worth changing the path `/one` to `/transaction` or something.
If `OneComponent` isn't being used anymore we can probably delete it.

Reply:
Same with `TwoComponent`!